opt/start-notebook.sh

```diff
@@ -9,7 +9,7 @@ export SHELL=/bin/bash
 
 # Enter home folder and start jupyterhub-singleuser.
 cd /home/${SYSTEM_USER}
-/opt/aiidalab-singleuser                                           \
+/opt/conda/bin/python /opt/aiidalab-singleuser                                           \
```

Comment:
is the conda python not the default python executable?

i.e. shouldn't  `#!/usr/bin/env python` resolve to `/opt/conda/bin/python`?

Reply:
same as [here](https://github.com/aiidalab/aiidalab-docker-stack/pull/36/files#r376408695)